clients/python-apiserver-client/python_apiserver_client/kuberay_apis.py

```diff
@@ -0,0 +1,261 @@
+import requests
+import time
+
+from .params import *
+
+
+_headers = {"Content-Type": "application/json", "accept": "application/json"}
+
+
+class KubeRayAPIs:
+    """
+        This class implements KubeRay APIs based on the API server.
+        To create a classe, the following parameters are required:
+            base - the URL of the API server (default is set to the standalone API server)
+            wait interval - the amount of sec to wait between checking for cluster ready
+    """
+    def __init__(self, base: str = "http://localhost:8888",
```

Comment:
Even though you provide docs for the arguments, how about more explicit names, like `server_url` and `wait_secs_for_server_ready`. Verbose, but obvious.